test/test_transforms_v2_consistency.py

```diff
@@ -929,6 +929,29 @@ def test_randaug(self, inpt, interpolation, mocker):
 
             assert_close(expected_output, output, atol=1, rtol=0.1)
 
+    @pytest.mark.parametrize(
+        "interpolation",
+        [
+            v2_transforms.InterpolationMode.NEAREST,
+            v2_transforms.InterpolationMode.BILINEAR,
+        ],
+    )
+    def test_randaug_jit(self, interpolation):
```

Comment:
@vfdev-5 @pmeier instead of adding new tests here, shouldn't we be writing the new ones in `test_transforms_v2_refactored.py`?

We're going to delete this file, right?

Reply:
@NicolasHug can we merge this PR and address your comment later ?

Reply:
> We're going to delete this file, right?

Yes. Indeed, all new tests should go into `test_transforms_v2_refactored.py`. No strong opinion of doing this now or later.

Reply:
OK, let's do it later